docs/jade-getting-started.md

```diff
@@ -217,12 +217,22 @@ git clone https://github.com/broadinstitute/datarepo-helm
 git clone https://github.com/broadinstitute/datarepo-helm-definitions
 ```
 
-Create your datarepo helm definition:
-1. In `datarepo-helm-definitions/dev` directory, copy an existing developer
+## 8. [DRAFT] Set up your Development Environment
+Note: These instructions have not been tested yet! This may be a good step to pair on with another jade team member. There is a video of us walking through these steps in our [Jade Googe Drive Folder](https://drive.google.com/drive/folders/1JM-_M0qsX6eXocyPc9TB7ivCKJTji3dX?usp=sharing). 
+
+1. Follow [instructions in our terraform-jade repository](https://github.com/broadinstitute/terraform-jade/tree/master/old#new-team-member-process) to add your initials to the terraform templates and generate the static resources needed to deploy your personal development environment. Apply the changes and create a pull request to merge your additons to terraform-jade.
+2. Create your datarepo helm definition:
+  -  In `datarepo-helm-definitions/dev` directory, copy an existing developer
 definition and change all initials to your own.
-2. Create a pull request with these changes in
-[datarepo-helm-definitions](https://github.com/broadinstitute/datarepo-helm-definitions)
-3. Ask a colleague from DevOps to create a service account and database for your deployment.
+  -  Create a pull request with these changes in [datarepo-helm-definitions](https://github.com/broadinstitute/datarepo-helm-definitions)
+3. Connect to your new dev postgres database instance (replace ZZ with your inintials):
+Note that this is separate instance than the local one you will configure in step 9. The following command connected to the database via a proxy.
+```
+cd jade-data-repo/ops
+DB=datarepo SUFFIX=ZZ ENVIRONMENT=dev ./db-connect.sh
+```
+4. Add some setting to your database, that I can't remembmer... 
```

Comment:
I just requested access to the recording on our session from Monday - I'll update this after watching that!